src/view/plandetail/PlacePreview.tsx

```diff
@@ -91,6 +91,23 @@ export const PlacePreview = ({
         onUpdateLikeAtPlace?.({ like: !like, placeId });
     };
 
+    const handleFileInput = (event: Event) => {
+        const target = event.target as HTMLInputElement;
+        const file = target.files && target.files[0];
```

Comment:
あるのかわからないけど、選択されたファイルが0件の場合もあるかもだから
要素数の検証もつけたほうが良さそう 🤔 
```suggestion
        const file = target.files && target.files.length > 0 &&target.files[0];
```